test/integration/access.go

```diff
@@ -74,10 +74,17 @@ func (a *accessLogs) check(infra *infra) error {
 
 				pod := infra.apps[app][0]
 				container := inject.ProxyContainerName
-				if app == "mixer" {
+				ns := infra.Namespace
+				switch app {
+				case "mixer":
 					container = "mixer"
+					ns = infra.IstioNamespace
```

Comment:
How is this test going to pass for multiple people if they all use same namespace?